### PR TITLE
Add unit test for `ViewCommand()`

### PR DESCRIPTION
**Pull request by Sweetdevil144**
This PR is a part of Issue #144 which aims to add tests for `project/view.go`

### CODE REVIEW

File: test/e2e/test.md [hunk 1]
```diff
@@ -1 +1,3 @@
-e2e tests for cmd can slide in here
\ No newline at end of file
+# Test.md
+
+e2e tests for cmd can slide in here.
```
Comment (bupd) [inline test/e2e/test.md:3]:
```suggestion

```

remove this.

**Review by bupd — CHANGES_REQUESTED** [review]
This will happen and tests will fail eventually.

```
=== RUN   TestViewCommandOutput
time="2024-08-13T23:55:01+05:30" level=error msg="failed to get project: response status code does not match any response statuses defined for this endpoint in the swagger spec (status 404): {}"
    view_test.go:85:
                Error Trace:    /home/bupd/code/OSS/PRCHECKcli/test/project/view_test.go:85
                Error:          Received unexpected error:
                                unexpected end of JSON input
                Test:           TestViewCommandOutput
    view_test.go:87:
                Error Trace:    /home/bupd/code/OSS/PRCHECKcli/test/project/view_test.go:87
                Error:          Not equal:
                                expected: map[string]interface {}{"Payload":map[string]interface {}{"creation_time":"2024-08-03T07:51:27.739Z", "current_user_role_ids":interface {}(nil), "cve_allowlist":map[string]interface {}{"creation_time":"0001-01-01T00:00:00.000Z", "id":3, "items":[]interface {}{}, "project_id":3, "update_time":"0001-01-01T00:00:00.000Z"}, "metadata":map[string]interface {}{"public":"false"}, "name":"my-project", "owner_id":4, "owner_name":"RolandK87", "project_id":3, "repo_count":1, "update_time":"2024-08-03T07:51:27.739Z"}}
                                actual  : map[string]interface {}(nil)
```

**Comment by bupd** [comment]
@Sweetdevil144 Make tests resilient to changes in demo.goharbor.io.

File: test/project/view_test.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "public": "false"
+    },
+    "name": "my-project",
+    "owner_id": 4,
+    "owner_name": "RolandK87",
+    "project_id": 3,
+    "repo_count": 1,
+    "update_time": "2024-08-03T07:51:27.739Z"
+  }
+}
```
[the remark this answers is not in the document]
Comment (Sweetdevil144) [reply test/project/view_test.json:22]: Should we add a 'demo' Project to be maintained only for the testing purpose by naming it 'test_project' so the contents are same in the future too while at the same time maintaining the checking?

**Pull request closed without merging**